editor/import/resource_importer_obj.cpp

```diff
@@ -249,6 +250,19 @@ static Error _parse_obj(const String &p_path, List<Ref<Mesh>> &r_meshes, bool p_
 			vtx.y = v[2].to_float() * scale_mesh.y + offset_mesh.y;
 			vtx.z = v[3].to_float() * scale_mesh.z + offset_mesh.z;
 			vertices.push_back(vtx);
+			//vertex color
```

Comment:
We use sentence case with punctuation for comments.

Is there a variation of vertex color has an alpha?

Reply:
> We use sentence case with punctuation for comments.

The nearby comments are `//vertex`, `//uv`, `//normal`, so I think it's consistent.

 > Is there a variation of vertex color has an alpha?

Per http://paulbourke.net/dataformats/obj/colour.html, the `v x y z r g b ` form does not have an alpha. There is a ZBrush-specific `#MRGB` extension that does have alpha.